x-pack/plugins/lens/public/shared_components/legend/layout/columns_number_setting.tsx

```diff
@@ -24,20 +24,20 @@ interface ColumnsNumberSettingProps {
   /**
    * Indicates if legend is located outside
    */
-  isLegendOutside: boolean;
+  isHidden: boolean;
```

Comment:
Is this prop still used to indicate if the legend is outside or did this change?

If it no longer impacted by/inside outside maybe we update the TSDocs.